static/js/message_store.js

```diff
@@ -58,6 +58,41 @@ exports.process_message_for_recent_subjects = function process_message_for_recen
     recent_subjects.set(stream, recents);
 };
 
+exports.process_message_for_recent_private_messages = function process_message_for_recent_private_messages(message, remove_message) {
+    var current_timestamp = 0;
+
+    if (recent_private_messages.length === 0)  {
+        recent_private_messages = [];
+    }
```

Comment:
What is this blocked needed for?  It looks like the global is initialized to [] already, so it would seem this shouldn't have any effect.